Library/ViewModels/ManagePledgeViewModelTests.swift

```diff
@@ -195,4 +199,160 @@ internal final class ManagePledgeViewModelTests: TestCase {
     self.goToUpdatePledgeProject.assertValues([Project.template])
     self.goToUpdatePledgeReward.assertValues([Reward.template])
   }
+
+  func testRewardReceivedViewControllerIsHidden_NoReward_Canceled() {
```

Comment:
These tests could potentially only test when the view is `NOT` hidden but since tests are cheap it should be not a problem for testing all the variations here.